drivers/display/ls0xx.c

```diff
@@ -0,0 +1,493 @@
+/*
+ * Copyright (c) 2020 Rohit Gujarathi
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT   sharp_ls0xx
+
+#define LOG_LEVEL CONFIG_DISPLAY_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_REGISTER(ls0xx);
+
+#include <string.h>
+#include <device.h>
+#include <drivers/display.h>
+#include <init.h>
+#include <drivers/gpio.h>
+#include <drivers/spi.h>
+#include <sys/byteorder.h>
+
+#include <display/cfb.h>
+
+/* Note:
+ * -> .write API should be called atleast once per second so that VCOM can 
+ *      be toggled. If there is nothing to update then buf can be set to NULL
+ *      or, width/height can be set to 0 so the driver will only 
+ *      toggle VCOM and return
+ * -> high/1 means white, low/0 means black
+ * -> Display expects LSB first
+ * -> If devicetree contains DISP pin then it will be set to high
+ *      during initialization.
+ * -> VCOM is toggled through software by default (CONFIG_LS0XX_EXTMODE_SERIAL=y)
+ *    EXTMODE pin is connected to VSS 
+ * -> To control VCOM using external signal EXTCOMIN, set
+ *      CONFIG_LS0XX_EXTMODE_SERIAL=n
+ *      EXTMODE pin is connected to VDD
+ * -> When designing with a memory LCD panel, a decision must be made as to how
```

Comment:
Seem to be 1:1 copy or the VCOM chapter from the appn, please remove it and add short description + reference to the appn/chapter.